Spring Boot - Backend/src/main/java/com/blog/example/FileUpload/Save/Controller/FileUploader.java

```diff
@@ -0,0 +1,72 @@
+package com.blog.example.FileUpload.Save.Controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.websocket.server.PathParam;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+@CrossOrigin(origins = "http://localhost:4200")
+@Controller
+public class FileUploader {
+
+    List<String> files = new ArrayList<String>();
```

Comment:
It should be `List<String> files = new ArrayList<>();`

Reply:
done